apps/form-app-e2e/tsconfig.e2e.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "sourceMap": false,
+    "outDir": "../../dist/out-tsc",
+    "allowJs": true,
+    "types": [
+      "cypress",
+      "node",
+      "cypress-axe"
+    ]
+  },
+  "include": [
+    "src/**/*.ts",
+    "src/**/*.js"
+  ]
+}
```

Comment:
remove space at the end of the file

Reply:
This file doesn't have empty space or empty line in the end.